packages/brick_supabase/lib/src/query_supabase_transformer.dart

```diff
@@ -186,9 +186,20 @@ class QuerySupabaseTransformer<_Model extends SupabaseModel> {
     final queryKey = (leadingAssociations != null ? '${leadingAssociations.join('.')}.' : '') +
         definition.columnName;
 
+    String queryValue;
+    if (condition.compare == Compare.inIterable) {
+      if (condition.value is Iterable && (condition.value as Iterable).isNotEmpty) {
+        queryValue = 'in.(${(condition.value as Iterable).join(',')})';
+      } else {
+        queryValue = 'in.()';
+      }
+    } else {
+      queryValue = '${_compareToSearchParam(condition.compare)}.${condition.value}';
+    }
+
     return [
       {
-        queryKey: '${_compareToSearchParam(condition.compare)}.${condition.value}',
+        queryKey: queryValue,
```

Comment:
The multiple `if/else` statements are overly complicated - it can be very simple
```suggestion
        queryKey: condition.compare == Compare.inIterable && condition.value is Iterable
            ? 'in.(${(condition.value as Iterable).join(',')})'
            : '${_compareToSearchParam(condition.compare)}.${condition.value}',,
```